scripts/getCountryCode.js

```diff
@@ -0,0 +1,20 @@
+(async function () {
+    const geoRestrictions = ["DE"];
+    try {
+        const res = await fetch("https://duckduckgo.com/country.json");
```

Comment:
Edge case: I'm wondering if we need a fetch polyfill?

```
# https://stackoverflow.com/questions/65094327/how-to-correctly-add-fetch-polyfill-to-be-used-by-separate-js-files
<script> 
  window.fetch || document.write('<script src="/fetch.min.js"><\/script>');
</script>
```

We could use grab a minified version of https://github.com/developit/unfetch?tab=readme-ov-file?